<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ngo_Sign_in</class>
 <widget class="QDialog" name="Ngo_Sign_in">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="ngosigninheader">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background :none;
color:red;
</string>
   </property>
   <property name="text">
    <string>Sign in as NGO</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="NgosigningroupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>361</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background :none;
color:red;
</string>
   </property>
   <property name="title">
    <string>Sign in Details</string>
   </property>
   <widget class="QLabel" name="Ngoregistration">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background :none;
color:red;
</string>
    </property>
    <property name="text">
     <string>Registration NO:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ngoregistrationNolineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background :none;
color:red;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="ngoname_signin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background :none;
color:red;
</string>
    </property>
    <property name="text">
     <string>Ngo Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ngoname_lineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background :none;
color:red;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ngo_password_lineedit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background :none;
color:red;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="ngo_signin_password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background :none;
color:red;
</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ngo_signin_pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background :none;
color:red;
</string>
    </property>
    <property name="text">
     <string>Sign in</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-image: url(&quot;C:/Users/hp/Desktop/DTE/return.png&quot;);
                     background-repeat: no-repeat;
                     background-position: center;
                     background-attachment: fixed;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
